packages/admin/src/common/toolbar/Toolbar.tsx

```diff
@@ -0,0 +1,22 @@
+import { Paper, Toolbar as MuiToolbar } from "@material-ui/core";
+import * as React from "react";
+
+import { MenuContext } from "../../mui";
+import { useStyles, useThemeProps } from "./Toolbar.styles";
+
+interface ToolbarProps {}
+const Toolbar: React.FunctionComponent<ToolbarProps> = ({ children }) => {
+    const themeProps = useThemeProps();
+    const { headerHeight } = React.useContext(MenuContext);
+    const classes = useStyles({ headerHeight });
+
+    return (
+        <Paper elevation={themeProps.elevation} className={classes.root}>
```

Comment:
I thinks you should do this, otherwise you might override all existing classNames:
```suggestion
        <Paper elevation={themeProps.elevation} classes={{ root: classes.root }}>
```